### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by yoooooooooooooooooooooooo** [coolwanglu/pdf2htmlEX#772]

**Comment by reckart** [comment]
See the first line in the README?

> pdf2htmlEX is no longer under active development. New maintainers are [wanted](http://pdf2htmlex.blogspot.ch/2016/12/looking-for-new-maintainer.html).

**Comment by yoooooooooooooooooooooooo** [comment]
Yes but it was added in 2016. Is there anybody maintaining it now?

**Comment by reckart** [comment]
No idea, but my guess would be: if there was somebody, they'd probably have removed that notice.

**Comment by Laim** [comment]
> Yes but it was added in 2016. Is there anybody maintaining it now?

By the looks of things, no.  Nothing has been committed since 2017.  Doesn't mean it's dead, just means no one has picked it up yet - could be revived easily if someone wanted to.
